index.js

```diff
@@ -281,3 +281,25 @@ for (let k = 0; k < project.length; k += 1) {
     createPopup(k);
   });
 }
+
+const form = document.querySelector('form');
+const mail = document.getElementById('useremail');
+const small = document.createElement('small');
+const errorMessage = 'Please enter email address in lowercase only.';
+
+function checkCase(input) {
+  return /[A-Z]/.test(input);
+}
+
+form.addEventListener('submit', (e) => {
+  e.preventDefault();
+
+  const mailInput = mail.value.trim();
+
+  if (checkCase(mailInput)) {
+    form.appendChild(small);
+    small.textContent = errorMessage;
+  } else {
+    form.submit();
+  }
+});
```

Comment:
[OPTIONAL] I think we could wrap all of our JS code in a `main` function. And the call `main` once the web app is finished loading. This will ensure that it works correctly. Imagine that the script loads, but maybe some of the elements on the document aren't still ready. Then it could fail to select these.

Start the script running once the page has finished loading like this: `window.onload = main`